SpheriCart.jl/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
I think it's a bit silly to have different licenses on different parts of the code. Would you be fine with dual-licensing everything as Apache/MIT? This way we have the same license for all the languages, and downstream user can use the code according to either license.

Reply:
I think that should be ok. At least I don't see any problematic consequences.

Reply:
Are you going to add Apache/MIT globally to the repo? Once you do this we can merge it into the PR and I could then remove the MIT license I guess?

Reply:
yes, that sounds like a plan!